backend/app/db/supabase/users_service.py

```
@@ -0,0 +1,205 @@
+            "verification_token_expires_at": expiry_time.isoformat(),
+            "updated_at": datetime.now().isoformat()
+        }).eq("discord_id", discord_id).execute()
+
+        if update_res.data:
+            _verification_sessions[session_id] = (discord_id, expiry_time)
+            logger.info(
+                f"Created verification session {session_id} for Discord user {discord_id}, expires at {expiry_time}")
+            return session_id
+        else:
+            logger.error(f"Failed to set verification token for Discord ID: {discord_id}. User not found.")
+            return None
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)